requirements.txt

```diff
@@ -2,3 +2,4 @@ azure-batch==3.0.0
 azure-storage==0.33.0
 pytest==3.1.3
 pylint==1.7.2
+pyyaml==3.12
```

Comment:
Just because we may need to think about this (way?) down the road, should we have a pattern where we split this file up into development and runtime sections?

```text
# production
azure-storage==0.33.0
...

# development
pytest=...
pylint=...
```

Honestly I have no clue if this is even supported in requirements files though...

Reply:
yeah you can add comments in the file https://github.com/pypa/pip/issues/564